### ISSUE TITLE
Toolbar - Always tabs to the first item

### ISSUE THREAD

**Issue by ling1726** [microsoft/fluentui#30111]
Currently the toolbar will always focus on the first item when a user tabs into it. This results in a bad experience for keyboard users especially since the toolbar doesn't have 'n of m' narration.

The toolbar should always remember the last focused item and move focus into that
